addons/sale_timesheet/views/sale_timesheet_portal_templates.xml

```diff
@@ -114,11 +114,11 @@
                 <div t-if="task.sale_order_id.invoice_ids"><strong>Invoices:</strong>
                     <span t-foreach="task.sale_order_id.invoice_ids" t-as="invoice_line">
                         <t t-if="invoice_line.id in invoices_accessible">
-                            <a t-attf-href="/my/invoices/{{ invoice_line.id }}">
+                            <a class="mr-1" t-attf-href="/my/invoices/{{ invoice_line.id }}">
                                 <i t-if="invoice_line.state == 'draft'">Draft Invoice</i>
                                 <t t-else="" t-esc="invoice_line.name"/>
                             </a>
-                            <span t-if="not invoice_line_last">,</span>
+                            <span class="mr-1" t-if="not invoice_line_last">,</span>
```

Comment:
Why is it needed? 🤔

Reply:
@xavierbol , Done all the changes. and there is underline till `,` when we have multiple invoices due to extra space added by </a> in the new line so updated it.  
Ref: https://aju.nimbusweb.me/share/6926917/ow0uc5ggyt3fu84ryops

Thanks